controllers/controller.js

```diff
@@ -16,4 +19,19 @@ function getEndpoints(req, res, next) {
   res.status(200).send({ endpoint });
 }
 
-module.exports = {getTopics, getEndpoints}
\ No newline at end of file
+function getArticleById(req, res, next) {
+    const articleArr = req.params
+    return fetchArticleId(articleArr.article_id)
+    .then((articles) => {
+        res.status(200).send({articles: articles})
```

Comment:
Getting a single article so would make more sense if this were on a key of `article`